contents/blog/improving-posthog-deployments.md

```diff
@@ -0,0 +1,104 @@
+---
+date: "2022-03-22"
+title: "Improving PostHog deployments"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/launch-week-teaser.jpeg # TODO
```

Comment:
@andyvan-ph @lottiecoxon do we have a launch week image we can use?

Reply:
Yup, I will add